kr8s/_api.py

```diff
@@ -187,8 +183,6 @@ async def open_websocket(
         if self.auth.token:
             headers["Authorization"] = f"Bearer {self.auth.token}"
         userauth = None
```

Comment:
Looks like the linter isn't happy about this variable being assigned and never used. We can remove it now.
```suggestion
```

Reply:
thanks for noticing it. Also, can you help me setup pre-commit locally so I can run the linter before PR submissions, please? I wish I had caught this locally. Eventually I amended the commit in the hope of fixing this but I'm not sure amends to the commit show up in the PR automatically (sorry, I am very used to the Gerrit code review workflow)

Reply:
You can install `pre-commit` with

```console
$ pip install pre-commit
```

Then add the hook in your local repo

```console
$ pre-commit install
```

Then when you make commits it will run the pre-commit hooks. 

```console
$ git commit -am "My awesome commit"
black....................................................................Passed
ruff.....................................................................Passed
[foo 124b7bc] My awesome commit
 1 file changed, 1 insertion(+)
```

You can also just run it manually with

```console
$ pre-commit run --all  
```

Reply:
Also just feel free to push more commits to this branch. Don't worry about amending or squashing. I will squash the PR into one commit when I merge it.